controller/user.go

```diff
@@ -134,6 +141,22 @@ func (c *Controller) IsLoggedIn(ctx context.Context) (bool, error) {
 	return isLoggedIn, nil
 }
 
+func confirmBrowserOpen(spinnerMsg string, url string) error {
```

Comment:
Function never technically returns an error right? Should just make it return nothing and save the err != nil check